src/packagedcode/windows_helpers.py

```diff
@@ -0,0 +1,311 @@
+import os
+from pathlib import PureWindowsPath
+
+import attr
+import logbook
+from regipy.exceptions import NoRegistrySubkeysException
+from regipy.exceptions import RegistryKeyNotFoundException
+from regipy.registry import RegistryHive
+
+from commoncode.command import execute
+from packagedcode import models
+
+
+logger = logbook.Logger(__name__)
+TRACE = False
+
+
+MSIINFO_BIN_LOCATION = 'packagedcode_msitools.msiinfo'
+
+
+def get_msiinfo_bin_location():
```

Comment:
Consider using a cascade of possible locations for this like in https://github.com/nexB/extractcode/blob/632f6a8b91bc7d81a6d5ab5743b34779d42c349d/src/extractcode/sevenzip.py#L66

Reply:
Done